Samples/FNSCandyStyleTransfer/UWP/cs/MainPage.xaml.cs

```diff
@@ -110,6 +114,39 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
 
             // Select first style
             UIStyleList.SelectedIndex = 0;
+
+            // Create a 1 second timer
+            _FramesPerSecondTimer.Tick += _FramesPerSecond_Tick;
+            _FramesPerSecondTimer.Interval = new TimeSpan(0, 0, 1);
+            _FramesPerSecondTimer.Start();
+        }
+
+        private void _FramesPerSecond_Tick(object sender, object e)
+        {
+            // how many seconds has it been?
+            int fpsTick = System.Environment.TickCount;
+            float numberOfSeconds = ((float)(fpsTick - _LastFPSTick)) / (float)1000;

```

Comment:
Seems to me like the first numberOfSeconds will be incorrect since _LastFPSTick is initialized to 0. 

However, since _LastFPSTick is updated on line 144, then it will be correct for future calls to _FramesPerSecond_Tick.

Reply:
fixed !